make_annot.py

```diff
@@ -29,8 +29,7 @@ def make_annot_files(args, bed_for_annot):
     df_annot.fillna(0, inplace=True)
     df_annot = df_annot[['ANNOT']].astype(int)
     if args.annot_file.endswith('.gz'):
-        with gzip.open(args.annot_file, 'wb') as f:
-            df_annot.to_csv(f, sep = "\t", index = False)
+        df_annot.to_csv(args.annot_file, sep = "\t", index = False, compression='\gzip')
```

Comment:
Can we get rid of the backslash on `\gzip`?

Reply:
also, pandas 0.24 can infer compression based on extension, but not in 0.20 which we're pinned to (bummer)